src/components/NavBar/navbar.tsx

```diff
@@ -0,0 +1,17 @@
+import { NavLink } from "react-router-dom";
+import useWalletConnect from "./connecttowallet";
```

Comment:
please use object definition of modules not default export

Reply:
The name of file should start with the capital letter `navbar.tsx` -> `Navbar.tsx`
https://www.upbeatcode.com/react/react-naming-conventions/